src/containers/home/Events.jsx

```diff
@@ -76,33 +77,43 @@ const Events = () => {
                 </div>
               ) : (
                 listToShow.map(
-                  ({ location, date, guests, name, link }, index) => {
+                  ({ location, date, guests, name, link, isPast }, index) => {
                     const info = [
                       `${dayjs(date).format('MMM DD, YYYY')} at ${dayjs(
                         date
-                      ).format('hh:mm a')}`,
-                      name
+                      ).format('hh:mm a')}`
                     ];
                     if (guests > 0) {
                       info.push(`${guests} guests`);
                     }
                     return (
                       <div
                         key={link}
-                        className={classnames('py-3', {
+                        className={classnames('py-3 relative', {
                           'border-b border-gray-200':
-                            index < fortcoming.length - 1
+                            index < listToShow.length - 1
                         })}
                       >
-                        <a
-                          href={link}
-                          target="_blank"
-                          rel="noopener noreferrer"
-                        >
-                          <H6 className="hover:text-primary transition-colors duration-200">
-                            {name}
-                          </H6>
-                        </a>
+                        {isPast && (
+                          <div className="absolute h-full w-full bg-foreground opacity-50 z-10 pointer-events-none" />
+                        )}
```

Comment:
what is the purpose of this `div`?

Reply:
To simulate the opacity 0.5 for a whole block without affect the stamp image's opacity
https://stackoverflow.com/questions/10021302/how-to-apply-an-opacity-without-affecting-a-child-element-with-html-css